ui/src/containers/BeamlineSetupContainer.jsx

```diff
@@ -323,10 +319,6 @@ function mapStateToProps(state) {
 
 function mapDispatchToProps(dispatch) {
   return {
-    getAllhardwareObjects: bindActionCreators(
```

Comment:
The unused action I mentioned was bound in this container but never dispatched from any of the child components (unless there's a weird prop access happening that I've missed, like `this.props['getAll' + 'hardwareObjects']`... :joy:).

Reply:
I don't think we are accessing any prop like that, other-tings perhaps but not props :)